raiden/transfer/state_change.py

```diff
@@ -520,7 +520,6 @@ def from_dict(cls, data: Dict[str, Any]) -> "ContractReceiveChannelClosedLight":
             canonical_identifier=CanonicalIdentifier.from_dict(data["canonical_identifier"]),
             block_number=BlockNumber(int(data["block_number"])),
             block_hash=BlockHash(deserialize_bytes(data["block_hash"])),
-            closing_participant=to_canonical_address(data["closing_participant"]),
```

Comment:
please leave a brief note about this change here

Reply:
added